core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/UniqueIDUserStoreManager.java

```diff
@@ -158,6 +158,25 @@ AuthenticationResult authenticateWithID(List<LoginIdentifier> loginIdentifiers,
      */
     List<User> getUserListOfRoleWithID(String roleName, String filter, int maxItemLimit) throws UserStoreException;
 
+
+    /**
+     * Get user list of group.
+     *
+     * @param groupName Name of the group.
+     * @return An array of users that belongs to the given group.
+     * @throws UserStoreException Thrown by the underlying UserStoreManager.
+     */
+    List<User> getUserListOfGroupWithID(String groupName) throws UserStoreException;
```

Comment:
Add it as a default method

Reply:
Resolved with https://github.com/wso2/carbon-kernel/pull/3951/commits/d9ab31ab25d4af10cbfc123f81cf6cfa4499734f